File: slack-reader/src/test/java/dev/number6/slackreader/generate/ChannelsListResponseGenerator.java
package dev.number6.slackreader.generate;

import dev.number6.slackreader.model.ChannelsListResponse;
import uk.org.fyodor.generators.Generator;
import uk.org.fyodor.range.Range;

public class ChannelsListResponseGenerator implements Generator<ChannelsListResponse> {

    @Override
    public ChannelsListResponse next() {
        return new ChannelsListResponse(Boolean.TRUE, SlackReaderRDG.channels(Range.closed(5,20)).next());
    }
}

File: slack-reader/src/test/kotlin/dev/number6/slackreader/generate/ChannelsListResponseGenerator.kt
package dev.number6.slackreader.generate

import dev.number6.slackreader.model.ChannelsListResponse
import uk.org.fyodor.generators.Generator
import uk.org.fyodor.range.Range
import java.lang.Boolean

class ChannelsListResponseGenerator : Generator<ChannelsListResponse> {
    override fun next(): ChannelsListResponse {
        return ChannelsListResponse(Boolean.TRUE, SlackReaderRDG.channels(Range.closed(5, 20)).next())
    }
}